src/sql/src/statements/copy.rs

```diff
@@ -44,6 +52,52 @@ impl CopyTable {
         &self.format
     }
 }
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct CopyTableFrom {
+    table_name: ObjectName,
+    format: Format,
+    connection: Option<HashMap<String, String>>,
+    pattern: Option<String>,
+    from: String,
+}
+
+impl CopyTableFrom {
+    pub(crate) fn new(
+        table_name: ObjectName,
+        from: String,
+        format: Format,
+        pattern: Option<String>,
+        connection: Option<HashMap<String, String>>,
+    ) -> Self {
+        CopyTableFrom {
+            table_name,
+            format,
+            connection,
+            pattern,
+            from,
+        }
+    }
```

Comment:
Maybe we could just pub all fields so we don't need a constructor and so many getters.